### PR TITLE
added an option to show unhandled errors on the console

### LINKED ISSUE #94: Print error.stack on handled exceptions
For exceptions that are caught and manually sent to Rollbar, it would be nice if error stack trace was printed out. Or perhaps an option to enable that.

### PR DESCRIPTION
**Pull request by lehresman**
Resolves #94

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -259,6 +259,15 @@ Default: `['passwd', 'password', 'secret', 'confirm_password', 'password_confirm
 Default: `[]`
 </dd>
 
+<dt>showUnhandledErrors
+</dt>
+<dd>Whether or not to show unhandled errors on the console before sending them to Rollbar.  By default, unhandled errors will be consumed and silently sent to Rollbar. 
+
+Default: `false`
```
Comment (brianr) [inline README.md]: IMO this should default to `true`; it's weird to me that adding this library would make me have to change the way I debug things locally. But I'll defer that to you, @rokob and @coryvirok .

Comment (lehresman) [inline README.md]: When @coryvirok and I discussed this last week, he suggested a default of `false` to keep behavior the same.  Easy to change, I have no personal preference.

Comment (coryvirok) [inline README.md]: Couple of things:

- Is this already happening for uncaught errors? https://github.com/rollbar/node_rollbar/blob/master/rollbar.js#L264
- If an error is passed into `handleError()` it is done so either by directly calling `handleError()` by the user or via one of the uncaught error handlers. For both of those cases it's really up to the user to tell us to print something out. So if the caller is already doing this we wouldn't want to default to printing them out a second time. Hence, defaulting to `false`. @brianr

Comment (brianr) [inline README.md]: @coryvirok good points. Sounds like this should only affect handled errors? The (new) docs make it sound like this is for uncaught errors.

**Comment by rokob** [comment]
Why are we using console and not logger, i.e. `logger.error`? Do unhandled exceptions get logged twice?

**Comment by lehresman** [comment]
Thanks for pointing this out, @rokob.  I'll submit a fixed version shortly.

**Comment by lehresman** [comment]
After talking about this in more depth with @rokob, we concluded the solution would be to show the error and stack trace for any manually-invoked message handler if this option is set to true.

File: lib/notifier.js [hunk 2]
```diff
@@ -26,6 +26,7 @@ var SETTINGS = {
   framework: 'node-js',
   root: null,  // root path to your code
   branch: null,  // git branch name
+  showUnhandledErrors: false, // show unhandled errors on the console before sending to Rollbar
```
Comment (rokob) [inline lib/notifier.js]: Gotta change to `showReportedMessageTraces` here as well.

Comment (lehresman) [inline lib/notifier.js]: Good catch.  Thanks.  Just pushed a fix.

**Pull request merged**